### PR TITLE
Enhanced visualization of MLS-Map

### PR DESCRIPTION
**Pull request by skasperski**
- Added new parameter to show only patches that have a minimum number of measurements.
- Always draw patches face-up (normal vector pointing upwards)

### CODE REVIEW

File: src/grid/SurfacePatches.hpp [hunk 1]
```diff
@@ -268,6 +268,11 @@ class SurfacePatch<MLSConfig::SLOPE> : public SurfacePatchBase
         return z_pos;
     }
 
+    int getNumerOfMeasurements() const
```
Comment (chhtz) [inline src/grid/SurfacePatches.hpp]: Typo: getNum**b**erOfMeasurements

File: viz/MLSMapVisualization.cpp [hunk 2]
```diff
@@ -617,8 +576,61 @@ void MLSMapVisualization::setSimplifySurface(bool enabled)
...
+void MLSMapVisualization::visualize(vizkit3d::PatchesGeode& geode, const SurfacePatch<MLSConfig::SLOPE>& p)
+{
+    float minZ, maxZ;
+    p.getRange(minZ, maxZ);
+    minZ -= 5e-4f;
+    maxZ += 5e-4f;
+    Eigen::Vector3f normal = p.getNormal();
+    if(normal.z() < 0)
+        normal *= -1.0;
+    if(p.getNumerOfMeasurements() >= minMeasurements)
```
Comment (chhtz) [inline viz/MLSMapVisualization.cpp]: I would suggest to break early `if(p.getNumerOfMeasurements() < minMeasurements) break;` before calculating normals, etc.

**Review by chhtz — CHANGES_REQUESTED** [review]

File: viz/MLSMapVisualization.cpp (hunk 2 above)
Comment (skasperski) [reply viz/MLSMapVisualization.cpp]: @chhtz done

Comment (chhtz) [reply viz/MLSMapVisualization.cpp]: Thanks!

**Pull request merged**